src/server/db/resources.js

```diff
@@ -0,0 +1,637 @@
+const topics = [
+  'Before you Start a Social Business',
+  'Starting a Social Business',
+  'Running a Social Business',
+  'Growing A Social Business',
+  'Case Studies',
+];
+
+const classes = [
```

Comment:
looking at this data we need to add some fields to the resources table:

title, form_title, form_url